docker-compose.yaml

```diff
@@ -36,7 +36,7 @@ x-django: &base_django
       DJANGO_MEDIA_ROOT: '/assets/media/'
       SENTRY_DSN: ${DJANGO_SENTRY_DSN}
       SENTRY_SAMPLE_RATE: ${DJANGO_SENTRY_SAMPLE_RATE:-0.2}
-      MAPSWIPE_ENVIRONMENT: ${MAPSWIPE_ENVIRONMENT:-dev}
+      MAPSWIPE_ENVIRONMENT: ${MAPSWIPE_ENVIRONMENT:-prod}
```

Comment:
Why the change to prod here? I would have thought that keeping things to dev by default would make it easier to start a dev env for newcomers, and also avoid any accidental actions on prod for users who happen to have prod info available.

Reply:
Currently **docker-compose.yaml** is not development friendly as we need to build for each change and also the nginx configs require some custom work. That is why we created **docker-compose.tc.yaml** to fit our development need which is also WIP.
So, as we are using **docker-compose.yaml** in the production run, I used the environment to be _prod_ by default. We should define this accordingly in staging/production. But also we are using separate sentry projects in staging/production, this is not really used.

Reply:
> I'm baffled by the fact that django doesn't support on delete constraints!

Same here.